pyproject.toml

```
@@ -1,30 +1,32 @@
-[tool.poetry]
+[project]
 name = "dac7"
 version = "2024.4.0"
 description = "Tooling for DAC7"
 readme = "README.md"
 license = "LICENSE.md"
```

```suggestion
license = { file = "LICENSE.md" }
```
https://python-poetry.org/docs/pyproject/#license